### DISCUSSION TITLE
Microsoft Learn에 연결된 repo의 main branch에 api-starter 폴더가 존재하지 않습니다

### DISCUSSION THREAD

**Discussion by angie4u** [devrel-kr/HackaLearn#115] [2 Q&A - 클라우드 스킬 챌린지]
![image](https://user-images.githubusercontent.com/16282358/128654212-5f7c98c0-cf97-4254-b5e2-b7ba35c5bb13.png)

* 관련하여 리포팅 된 이슈: https://github.com/MicrosoftDocs/mslearn-staticwebapp/issues/52
* 해결 방안: 
https://github.com/MicrosoftDocs/mslearn-staticwebapp 대신 https://github.com/MicrosoftDocs/mslearn-staticwebapp-api 리포 활용 

혹시 다른 분들도 어려움 겪으실까봐 기록용으로 남깁니다.

**Comment by jung-youjin** [comment]
@angie4u 우와아아💪🏻 좋은 자료 공유 감사합니다😊
